frontend/packages/dev-console/src/components/PodsRing.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { VictoryLabel } from 'victory';
+import * as _ from 'lodash';
+import {
+  K8sResourceKind,
+  k8sPatch,
+  K8sKind,
+  SelfSubjectAccessReviewKind,
+} from '@console/internal/module/k8s';
+import { AngleUpIcon, AngleDownIcon } from '@patternfly/react-icons';
+import { checkEditAccess } from '@console/internal/components/utils';
+import PodStatus from './topology/shapes/PodStatus';
+import { Pod } from './topology/topology-types';
+import SvgDefsProvider from './svg/SvgDefsProvider';
+import { calculateRadius } from './topology/shapes/WorkloadNode';
```

Comment:
This is a bad dependency. This should not depend on any util from a topology.
Refactor everything related to the pods visualization into a common folder and make this and the topology node depend on the new code.